src/libs/ReportUtils.js

```diff
@@ -460,6 +460,17 @@ function isConciergeChatReport(report) {
     return lodashGet(report, 'participantAccountIDs', []).length === 1 && Number(report.participantAccountIDs[0]) === CONST.ACCOUNT_ID.CONCIERGE && !isChatThread(report);
 }
 
+/**
+ * Returns true if we can share a task in this report
+ * @param {Object} report
+ * @returns {Boolean}
+ */
+
+function isExpensifyChatReport(report) {
```

Comment:
Let's also either - update this method so it only filters for chat reports 

OR

Update the method name since it checks all reports right now to something like isExpensifyOnlyParticipantInReport